x-pack/filebeat/input/default-inputs/inputs_other.go

```diff
@@ -31,6 +32,7 @@ import (
 func xpackInputs(info beat.Info, log *logp.Logger, store beater.StateStore) []v2.Plugin {
 	return []v2.Plugin{
 		azureblobstorage.Plugin(log, store),
+		azureeventhub.Plugin(log),
```

Comment:
I am a bit confused, what's this for?

Reply:
This registers `azureeventhub` (using the input plugin v2 API) and makes it available to Filebeat. Previous, the registration was happening using the old API:

https://github.com/elastic/beats/blob/db9406b50514b8896f08e7bbb176f09c07650566/x-pack/filebeat/input/azureeventhub/input.go#L35-L40

Without registration, the plugin is unknown to Filebeat.